src/plutus_gen/code_emitters_aiken.rs

```diff
@@ -468,16 +471,30 @@ pub fn emit_verifier_code(
         })
         .join("),(");
 
-    let commitment_map = format!("    let commitment_data = [({})]", commitment_data);
-    data.insert("COMMITMENT_MAP".to_string(), commitment_map);
+    let kzg_halo2_commitment_map =
+        format!("    let commitment_data = [({})]", halo2_commitment_data);
+    data.insert("COMMITMENT_MAP".to_string(), kzg_halo2_commitment_map);
 
-    let point_sets = unique_grouped_points
+    let kzg_halo2_point_sets = unique_grouped_points
         .iter()
         .map(|set| set.iter().map(decode_rotation).join(","))
         .join("],[");
 
-    let point_sets = format!("     let point_sets = [[{}]]", point_sets);
-    data.insert("POINT_SETS".to_string(), point_sets);
+    let kzg_halo2_point_sets = format!("     let point_sets = [[{}]]", kzg_halo2_point_sets);
+    data.insert("POINT_SETS".to_string(), kzg_halo2_point_sets);
```

Comment:
As far as I can see code generation for both GWC_KZG and Halo2_ZKG are mixed together.
How this works? You just rely on the absence of tags in particular templates? E.g., there will be no POINTS_SETS in GWC_KZG version, so this code won't be inserted anywhere?